.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,60 @@
+name: Maintenance 🧰
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  workflow_removal:
+    runs-on: ubuntu-latest
+    name: Workflow cleanup 🧹👷‍♂️
+    # Errors will probably be caused by excesses in API quota, so we can safely continue.
+    # Remaining workflows will be removed in the next scheduled run.
+    continue-on-error: true 
+    strategy:
+      fail-fast: false
+      matrix:
+        repo:
+          - 'lowlighter/metrics'
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2.3.4
+
+      - name: Prepare environment
+        run: chmod +x delete_workflows.sh
+        working-directory: ./maintenance
```

Comment:
Is this the right working directory? Should it `.github/workflows/maintenance`, like in the other job?

Also, couldn't this step be skipped completely if the executable file mode is tracked in the repo?
If you're on an OS that doesn't support this normally, like Windows, you can do 
```
git add --chmod=+x <filename>
```

Reply:
Good catch!

And thank you very much for the ``chmod`` trick, I wasn't aware of it.

Pushed a new commit with the changes.